ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -793,7 +934,8 @@ export const RuntimePanel = fp.flow(
 
   const {hasWgsData: allowDataproc} = findCdrVersion(cdrVersionId, cdrVersionTiersResponse) || {hasWgsData: false};
   let [{currentRuntime, pendingRuntime}, setRequestedRuntime] = useCustomRuntime(namespace);
-
+  const enablePD = serverConfigStore.get().config.enablePersistentDisk;
+  const {persistentDisk} = enablePD ? useStore(diskStore) : null;
```

Comment:
Hmm, I thought you mentioned we didn't need a store. Is it needed due to the lower level interactions (hooks, runtime initializer etc)?

Reply:
The reason we need the disk store is we can't get the PD information from runtime store after the runtime has been deleted. We need an isolated hook to retrieve the unattached disk state.